backends/platform/ios7/ios7_video.mm

```diff
@@ -228,6 +230,23 @@ - (void)setupGestureRecognizers {
 	[twoFingerLongPressGesture setDelaysTouchesEnded:NO];
 	[twoFingerLongPressGesture setCancelsTouchesInView:NO];
 	[twoFingerLongPressGesture canPreventGestureRecognizer:twoFingerTapGesture];
+	
+	pencilTouchGesture = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(pencilTouch:)];
+	[pencilTouchGesture setNumberOfTouchesRequired:1];
+	[pencilTouchGesture setAllowedTouchTypes:@[@(UITouchTypePencil)]];
+	[pencilTouchGesture setMinimumPressDuration:0];
+	[pencilTouchGesture setDelaysTouchesBegan:NO];
+	[pencilTouchGesture setDelaysTouchesEnded:NO];
+	[pencilTouchGesture setCancelsTouchesInView:NO];
+	
+	pencilThreeTapTouchGesture = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(pencilThreeTapTouch:)];
+	[pencilThreeTapTouchGesture setNumberOfTouchesRequired:1];
+	[pencilThreeTapTouchGesture setNumberOfTapsRequired:2];
```

Comment:
Wouldn't this configuration just require the user to do a double tap instead of three taps?

Reply:
It is three taps you have to make, but looking at this again I think technically it is two taps with the third tap actually being considered a touch by the gesture recogniser.

I can rename if you'd prefer, or keep it named as is matching the gesture that you're actually doing?